pulp_herminig/app/serializers.py

```diff
@@ -1,21 +1,68 @@
 from gettext import gettext as _
 
 from pulpcore.plugin.serializers import RelatedField
-from rest_framework.serializers import BooleanField, IntegerField, Serializer, ValidationError
+from rest_framework.serializers import (
+    BooleanField,
+    FloatField,
+    IntegerField,
+    Serializer,
+    ValidationError
+)
 
 
 class TaskingBenchmarkSerializer(Serializer):
-    truncate_tasks = BooleanField(default=False)
-    count = IntegerField(default=4)
-    resources_N = IntegerField(default=0)
-    resources_K = IntegerField(default=0)
+    truncate_tasks = BooleanField(
+        default=False,
+        help_text = _(
+            "If True, this will delete all final tasks from the database before dispatching."
+        ),
+    )
+    count = IntegerField(
+        default=4,
```

Comment:
```suggestion
        default=4,
        min_value=0,
```
makes sense here too.